datameta/views/admin.py

```diff
@@ -154,14 +150,13 @@ def v_admin_get(request):
     # If the requesting user is only a group admin, return only those users
     # that are in the same group
     if not req_user.site_admin:
-        query = query.filter(User.group_id==req_user.group_id)
+        query = query.filter(User.group.uuid==req_user.group_id)
 
     response = {}
 
     response["users"] = [ {
-        'uuid' : str(user.uuid),
-        'id' : user.site_id,
-        'group_id' : user.group.site_id,
+        'id' : {'uuid' : str(user.uuid), 'site_id' : user.site_id},
+        'group_id' : {'uuid' : str(user.group.uuid), 'site_id' : user.group.site_id},
```

Comment:
The entity schema is `{'uuid' : [...], 'site' : [...]}`
```suggestion
        'id' : {'uuid' : str(user.uuid), 'site' : user.site_id},
        'group_id' : {'uuid' : str(user.group.uuid), 'site' : user.group.site_id},
```